app/controllers/insured/group_selection_controller.rb

```diff
@@ -322,7 +347,12 @@ def revise_aptc_applied_total(params, enrollment_id)
   end
 
   def calculate_elected_aptc_pct(aptc_applied_amount, aggregate_aptc_amount)
-    (aptc_applied_amount / aggregate_aptc_amount).round(2)
+    return 0.00 if aggregate_aptc_amount.zero?
+    if params[:bs4]
+      (aptc_applied_amount / aggregate_aptc_amount).round(4)
+    else
+      (aptc_applied_amount / aggregate_aptc_amount).round(2)
+    end
```

Comment:
these changes were made just to handle more variance and the edge case of $0:

- if $0.00 was entered previously, it would error on plan reinstatement and would just terminate the plan
- if someone wants to enter a flat $20 as their aptc credit, we don't want it to be changed due to the percent coming out to be 0.10 vs what it should be, 0.1025